docs/node/RPC Category/16-loan.md

```diff
@@ -330,6 +330,48 @@ interface loan {
 }
 
 interface VaultDetails {
+  vaultId?: string
```

Comment:
@Jouzo `vaultId` is currently not returned by `getvault` RPC. should we add it there? one have to know the `vaultId` to call `getvault`. But should we add it for consistency?